docs/tutorials/09_Protein_Folding.ipynb

```diff
@@ -0,0 +1,824 @@
+{
+ "cells": [
+  {
+   "cell_type": "markdown",
+   "id": "739993c7",
+   "metadata": {},
+   "source": [
+    "# Protein Folding"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "d64f8162",
+   "metadata": {},
+   "source": [
+    "### Introduction"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "3bd209d1",
+   "metadata": {},
+   "source": [
+    "The structure and function of many natural and human-engineered\n",
+    "proteins is still only poorly understood. As a result, our understanding of\n",
+    "processes connected with protein folding, such as those encountered in\n",
+    "Alzheimer’s disease, vaccine development, and crop improvement\n",
+    "research, has remained limited.\n",
+    "\n",
+    "Unfolded polypeptides have a very large number of degrees of freedom\n",
+    "and thus an enormous number of potential conformations. For example, a\n",
+    "chain with 100 aminoacids has on the order of 10<sup>47</sup> conformations. In\n",
```

Comment:
The superscript does not get rendered properly in Qiskit's documentation html output:
![screenshot_1629098518](https://user-images.githubusercontent.com/21973473/129526137-187146de-da10-4349-8cc7-d36b32bbd0d6.png)